src/pages/[dtags].tsx

```diff
@@ -0,0 +1,9 @@
+import ProfileDetails from '@screens/profile_details';
+
+const ProfileDetailsPage = () => {
+  return (
+    <ProfileDetails />
+  );
+};
+
+export default ProfileDetailsPage;
```

Comment:
I made a mistake, file should be named `src/pages/[dtag].tsx` because this page shows a singular dtag profile not multiple